addons/website/views/snippets/s_title.xml

```diff
@@ -4,32 +4,11 @@
 <template id="s_title" name="Title">
     <section class="pt32 pb32">
         <div class="container">
-            <div class="row s_nb_column_fixed">
-                <div class="col-lg-12 s_title pt16 pb16" style="text-align: center;">
-                    <h1 class="s_title_default"><font style="font-size: 62px;">Your Site Title</font></h1>
-                </div>
-            </div>
+            <h1 class="s_title_default" style="text-align: center;"><font style="font-size: 62px;">Your Site Title</font></h1>
```

Comment:
`s_title_default` still exists?

Reply:
It's used in a few places in XML but there are no CSS rules targeting it anywhere (not even in themes). Might be targeted by custom css though. Should I remove it completely?

Reply:
Yes, I suppose it can be removed from everywhere